airbyte-cdk/python/airbyte_cdk/utils/airbyte_secrets_utils.py

```diff
@@ -67,5 +67,6 @@ def filter_secrets(string: str) -> str:
     # TODO this should perform a maximal match for each secret. if "x" and "xk" are both secret values, and this method is called twice on
     #  the input "xk", then depending on call order it might only obfuscate "*k". This is a bug.
     for secret in __SECRETS_FROM_CONFIG:
-        string = string.replace(str(secret), "****")
+        if secret:
```

Comment:
could you unit test this behavior please?

Reply:
ok I will add now